projects/igniteui-angular/src/lib/navigation-drawer/themes/shared/_bootstrap.scss

```
@@ -0,0 +1,26 @@
+@use 'igniteui-theming/sass/typography' as *;
+@use 'igniteui-theming/sass/bem' as *;
+@use 'styles/themes/standalone' as *;
+
+@include themed-block(igx-nav-drawer, bootstrap) {
+    --aside-padding: #{rem(16px)};
+    --item-padding: #{rem(8px) rem(16px)};
+    --item-gap: #{rem(8px)};
+    --item-min-height: #{rem(40px)};
+    --igx-nav-drawer-size--mini: #{rem(88px)};
+
+    @include e(item) {
+        @include type-style('body-2');
+    }
+
+    @include e(item, header) {
+        @include type-style('caption');
+    }
+
+    @include e(aside, mini) {
+        .igx-nav-drawer__item,
+        .igx-nav-drawer__item--active {
```

```suggestion
        .igx-nav-drawer__item,
        .igx-nav-drawer__item--active {
```
The same issue we had with the list item component. The item should be its own block, i.e. igx-nav-drawer-item. Then the modifiers (`active`, `header`, `disabled`, etc.) should be applied on the item block. This will allow you to not use the syntax in this way.